### PR TITLE
Add font size option

### PR DESCRIPTION
**Pull request by johnbintz**
This allows configuring the font output size of the rendered source code. It doesn't change the size of window decorations.

### CODE REVIEW

**Comment by matsuyoshi30** [comment]
@johnbintz Thanks for contribution and sorry for late response. I'll take a look later!

File: cmd/germanium/testdata/font-size.png
Comment (matsuyoshi30) [inline cmd/germanium/testdata/font-size.png:1]: It looks that it is necessary to revisit the calculation of the length of the string to be drawn.

Comment (matsuyoshi30) [reply cmd/germanium/testdata/font-size.png:1]: I think we should calculate the width length needs to take into account the display of line numbers here.

https://github.com/matsuyoshi30/germanium/blob/b6284068d6d50543f0635e8182bd5a9b02caa4e4/panel.go#L87

Comment (johnbintz) [reply cmd/germanium/testdata/font-size.png:1]: Done, I fixed that up and shuffled around some font size processing in the next commit.

**Comment by matsuyoshi30** [comment]
It looks great to me. Thank you @johnbintz !

**Pull request merged**